<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="gb_mode_avto">
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QRadioButton" name="rb_mode_auto">
         <property name="text">
          <string>Autonomous mode of traffic lights.</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_mode_slave">
         <property name="text">
          <string>Slave mode of traffic lights in RS 485.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="slave_widget" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lb_direction">
            <property name="text">
             <string>Set direction:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbDirect">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Direction 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 13</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Direction 16</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_read">
         <property name="text">
          <string>Read</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_send">
         <property name="text">
          <string>Write</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionConnection"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuSetting"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/ico/running.ico</normaloff>:/ico/running.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConnection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/ico/tools.ico</normaloff>:/ico/tools.ico</iconset>
   </property>
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/ico/traffic.ico</normaloff>:/ico/traffic.ico</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>rb_mode_slave</sender>
   <signal>toggled(bool)</signal>
   <receiver>slave_widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
